src/gtimelog/secrets.py

```diff
@@ -0,0 +1,219 @@
+"""
+Keyring and secrets
+"""
+import functools
+import logging
+from gettext import gettext as _
+
+from .utils import require_version
+
+require_version('Gtk', '3.0')
+require_version('Soup', '2.4')
+require_version('Secret', '1')
+from gi.repository import Gio, Gtk, GObject, Soup, Secret
+
+
+log = logging.getLogger('gtimelog.secrets')
```

Comment:
Maybe it's time to switch to the traditional `logging.getLogger(__name__)`?

Reply:
Indeed!